docs/rules/lines-around-directive.md

```diff
@@ -1,5 +1,7 @@
 # require or disallow newlines around directives (lines-around-directive)
 
+This rule was **deprecated** in ESLint v3.19.0 and replaced by the [padding-line-between-statements](padding-line-between-statements.md) rule.
```

Comment:
This will need to be updated from "ESLint v3.19.0" to the version that this PR gets merged in.